### DISCUSSION TITLE
A classic pure emerald is what color?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#464] [Galaxy Trivia] [answered]
*Answer in #RRGGBB*

**Comment by ParanoidUser** [comment]
@aliabdallah7, do you know? 😊

**Comment by aliabdallah7** [comment, marked as answer]
#50C878

**Reply by ParanoidUser** [reply]
Bingo! One more to go for the *Galaxy Brain* badge. 😊

**Comment by peterbaccaro** [comment]
A classic pure emerald is **Emerald Green** which has a hex code of **#50C878**.
